src/Compilers/CSharp/Test/Syntax/LexicalAndXml/RawStringLiteralLexingTests.cs

```diff
@@ -0,0 +1,284 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.LexicalAndXml
+{
+    public class RawStringLiteralLexingTests : CSharpTestBase
```

Comment:
I see `CreateCompilation` in this file too. I'd suggest moving all the new tests together into a file (similar to `RecordTest`).

Reply:
done.